apps/src/lib/levelbuilder/CourseVersionPublishingEditor.jsx

```diff
@@ -3,17 +3,13 @@ import React, {Component} from 'react';
 import HelpTip from '@cdo/apps/lib/ui/HelpTip';
 import color from '@cdo/apps/util/color';
 
-const publishedStates = ['Pilot', 'Beta', 'Preview', 'Recommended'];
+const PUBLISHED_STATES = ['pilot', 'beta', 'preview', 'recommended'];
```

Comment:
This is defined a few times, could it be added to `apps/src/lib/levelbuilder/contants.js`?